deploy-utils/maintenance/run-job.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Doesn't this need to check the directory? Similar to the other *.sh files.

Reply:
Oh, good catch, I had forgotten about that change to the other helper scripts.